### ISSUE TITLE
consider storing "raw selector" of locator to be able to log it as it is

### ISSUE THREAD

**Issue by yashaka** [yashaka/selene#438]
It will be possible to define custom selectors in Selene soon.

E.g. you will be able to configure Selene to search by text, when somebody wraps words into `«Here»`, for example the `browser.element('«Create document»')` will be translated automatically to `browser.element('//*[text()="Create document"]')`. 

The goal of this issue is to ensure, that use is able to see in log the original `«Create document»` selector, if he wants (yet in error message, he should be able to see the actual transpiled selector)

**Comment by aleksandr-kotlyar** [comment]
@yashaka I don't think that `browser.element('"Create document"')` will be simple to understand what it under hood. It's better to use
`browser.element(by.text('todos'))`. But may be I don't understand something, and you used this just like example to reach the goal of two-level logging, not implementing `browser.element('"Create document"')` directly 🙂

**Comment by yashaka** [comment]
It was just an example)

But your example is out of the scope of this ticket. This ticket is about
custom selector strategy inside a string, not outside). That's why
by.text('foo') is out of scope, but something like 'text=foo' is in scope.
The last one is exactly what is used in playwright, so should pretty
understandable) Nevertheless it is for the end user to choose how to
customize his selector strategy.
